examples/scanner.ts

```diff
@@ -17,12 +17,14 @@ import { platform } from 'os';
 
 import { scanner } from '../lib/';
 
-async function main() {
+const bootImageFolder = String(process.argv.slice(2));
```

Comment:
nitpick: You've changed the rest to `bootImageDir` so leaving this one even if local feels strange. I'd rather have it the same so if I grep for it I get all hits.